docs/guides/monorepo-react.md

```diff
@@ -0,0 +1,177 @@
+│   │   │   ├── plugins/
+│   │   │   └── support/
+│   │   ├── cypress.json
+│   │   ├── tsconfig.e2e.json
+│   │   └── tslint.json
+├── libs/
+├── nx.json
+├── package.json
+├── tools/
+├── tsconfig.json
+└── tslint.json
+```
+
+Run:
+
+- `ng serve frontend` to serve the application
+- `ng build frontend` to build the application
+- `ng lint frontend` to lint the application
+- `ng test frontend` to test the application using Jest
+- `ng e2e frontend-e2e` to test the application using Cypress
+
+As with Angular and Node, when using React in Nx, you get the out-of-the-box support for TypeScript, Cypress, Jest. No need to configure anything: watch mode, source maps, and typings just work.
+
+## Generating a React Library
+
+Run
+
+```bash
+ng g lib home --framework=react
+```
+
+and you will the following:
+
+```treeview
+myworkspace/
+├── README.md
+├── angular.json
+├── apps/
+│   ├── frontend/
+│   └── frontend-e2e/
+├── libs/
+│   └── home/
+│       ├── src/
+│       │   ├── lib/
+│       │   │    ├── home.css
+│       │   │    ├── home.tsx
+│       │   │    └── home.spec.tsx
+│       │   └ index.ts
+│       ├── jest.config.js
+│       ├── tsconfig.json
+│       ├── tsconfig.lib.json
+│       ├── tsconfig.spec.json
+│       └── tslint.json
+├── nx.json
+├── package.json
+├── tools/
+├── tsconfig.json
+└── tslint.json
+```
+
+Run:
+
+- `ng test home` to test the library
+- `ng lint home` to lint the library
+
+## Using Library in Application
+
+You can import the home library into the frontend application like this.
+
+```typescript
+import { Component } from 'react';
+import { Home } from '@myworkspace/home';
+
+import './app.css';
+
+export class App extends Component {
+  render() {
+    return (
+      <>
+        <Home />
+        <div>rest of app component</div>
+      </>
+    );
+  }
+}
+```
+
+## Sharing Code
+
+Without Nx creating a new shared library can take weeks: a new repo needs to be provisioned, CI needs to be set up, etc.. In an Nx Workspace it takes minutes.
+
+You can share React components between multiple React applications. You can share web components between React and Angular applications. You can share code between the backend and the frontend. All can be done without any unnecessary ceremony.
+
+## Understanding Your Nx Workspace
+
+An Nx workspace can contain dozens (or hundreds) of applications and libraries. It can be difficult to understand how they depend upon each other, and what are the implications of making a particular change.
+
+Previously, some senior architect would create an ad-hoc dependency diagram and upload it to a corporate wiki. The diagram isn’t correct even on Day 1, and gets more and more out of sync with every passing day.
+
+With Nx, you can do better than that. You can run `yarn dep-graph` to see a current dependency diagram of the workspace: what apps and libs are there, how they depend on each other, what is loaded lazily and what is not. Nx uses code analysis to collect this information.
```

Comment:
I don't think lazy loading is analyzed in React applications.. yet

Reply:
It is not. I'd still leave it in because it is a generic statement of types of things Nx can do.